src/Features.php

```diff
@@ -184,7 +184,11 @@ private static function get_fields(): array {
 	public static function is_active( string $name ): bool {
 		$features = get_option( Settings::KEY );
 
-		return isset( $features[ $name ] ) && $features[ $name ];
+		$active = isset( $features[ $name ] ) && $features[ $name ];
+
+		// Filter to allow setting a feature from code, to avoid chicken and egg problem when releasing adaptions to a
+		// new feature.
+		return (bool) apply_filters( "planet4_feature_$name", $active );
```

Comment:
We can think about which one should take precedence.

As it is now this would prevent from disabling the feature. This may actually not be a problem because the use case is where the new changes depend on the feature being active, and they were also tested on dev sites.